account_move_name_sequence/models/account_journal.py

```diff
@@ -0,0 +1,249 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# Copyright 2022 Vauxoo (https://www.vauxoo.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# @author: Moisés López <moylop260@vauxoo.com>
+# @author: Francisco Luna <fluna@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    sequence_id = fields.Many2one(
+        "ir.sequence",
+        string="Entry Sequence",
+        copy=False,
+        check_company=True,
+        domain="[('company_id', '=', company_id)]",
+        help="This sequence will be used to generate the journal entry number.",
+    )
+    refund_sequence_id = fields.Many2one(
+        "ir.sequence",
+        string="Credit Note Entry Sequence",
+        copy=False,
+        check_company=True,
+        domain="[('company_id', '=', company_id)]",
+        help="This sequence will be used to generate the journal entry number for refunds.",
+    )
+    # Redefine the default to True as <=v13.0
+    refund_sequence = fields.Boolean(default=True)
+
+    @api.constrains("refund_sequence_id", "sequence_id")
+    def _check_journal_sequence(self):
+        for journal in self:
+            if (
+                journal.refund_sequence_id
+                and journal.sequence_id
+                and journal.refund_sequence_id == journal.sequence_id
+            ):
+                msg = (
+                    "On journal '%s', the same sequence is used as "
+                    "Entry Sequence and Credit Note Entry Sequence."
+                )
+                raise ValidationError(_(msg, journal.display_name))
```

Comment:
Variables can't be translated

Reply:
@frahikLV 

It is weird, the original source is not using a variable:
 - https://github.com/OCA/account-financial-tools/blob/a22dbbba6c2b06e806ee4925b4398f5e089190a8/account_move_name_sequence/models/account_journal.py#L47-L52

@frahikLV why this change is required for this PR?

Could we do minimal changes here, please, in order to avoid diverging a lot from the original branch?